src/Compiler/CompilerOptions.cpp

```diff
@@ -601,6 +603,8 @@ bool parseCustomEnvFlagsCommandLineOption(
       envVar = arg.substr(sizeof("-customEnvFlags"));
       break;
     }
+    if (arg.compare("-v") == 0)
+      verbose = true;
```

Comment:
The arguments are scanned backwards so if `-customEnvFlags` or `---customEnvFlags` is specified after `-v`, `verbose` won't be set.

Also, if `-v` is specified inside the env var, e.g., `ONNX_MLIR_FLAGS="-v -O0"`, one would expect to get the same output below.

Reply:
Good points which I can address.

Reply:
Its now working: Below and example of the `-v` caught from the env var.

```
ONNX_MLIR_FLAGS="-O0 -v" onnx-mlir -O3 -march=arm64 -parallel aiu-par-test-v2.mlir 
Onnx-mlir default options from 'ONNX_MLIR_FLAGS' are '-O0 -v'.
Onnx-mlir command: onnx-mlir -O0 -v -O3 -march=arm64 -parallel aiu-par-test-v2.mlir
[/Users/alexe/OM/atests/ops/] /Users/alexe/OM/llvm-project/build/bin/opt: opt -O3 --mtriple=arm64-apple-darwin23.2.0 --march=arm64 --code-model small -o aiu-par-test-v2.bc aiu-par-test-v2.unoptimized.bc
[/Users/alexe/OM/atests/ops/] /Users/alexe/OM/llvm-project/build/bin/llc: llc -O3 --mtriple=arm64-apple-darwin23.2.0 --march=arm64 --code-model small -filetype=obj -relocation-model=pic -o aiu-par-test-v2.o aiu-par-test-v2.bc
[/Users/alexe/OM/atests/ops/] /usr/bin/c++: c++ aiu-par-test-v2.o -o aiu-par-test-v2.so -shared -fPIC -L/Users/alexe/OM/onnx-mlir/build/Debug/lib -lcruntime -lompruntime
Shared library 'aiu-par-test-v2.so' has been compiled.
```